extensions/azurehybridtoolkit/notebooks/hybridbook/content/hadr/configure-failover.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "powershell",
+            "display_name": "PowerShell",
+            "language": "powershell"
+        },
+        "language_info": {
+            "name": "powershell",
+            "codemirror_mode": "shell",
+            "mimetype": "text/x-sh",
+            "file_extension": ".ps1"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Create and configure Failover Cluster\n",
+                "\n",
+                "## Description\n",
+                "\n",
+                "Notebook to walk through creating a failover cluster on on-premises SQL Server Database."
+            ],
+            "metadata": {
+                "azdata_cell_guid": "6af59d69-ade7-480a-b33e-52a86fe5bfd3"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Note: before running the steps below, make sure you have the following:\n",
```

Comment:
I like to avoid using first person perspective and instead go for 3rd person. For example, remove any reference to "you" or "we" and just make a direct statement. "Note: before running the steps below, make sure to have the following:"

The "you" is implied but we're avoiding being personal.